tools/addbot.ts

```diff
@@ -21,6 +21,7 @@ const perms = flags.READ_MESSAGES |
   flags.EMBED_LINKS |
   flags.ATTACH_FILES |
   flags.READ_MESSAGE_HISTORY |
+  flags.MANAGE_MESSAGES |
```

Comment:
This doesn't look like it has anything to do with puppeting, so this would need to be in a seperate commit.

Reply:
Fair enough; I've taken it out.